product/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+from user.models import Allergy,User
+
+class Option(models.Model):
+    name = models.CharField(max_length=30, null=False)
+
+    class Meta:
+        db_table = 'options'
+
+class Category(models.Model):
+    name = models.CharField(max_length=30, null=False)
+
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    name = models.CharField(max_length=30, null=False)
+    category = models.ForeignKey(Category, on_delete=models.DO_NOTHING, null=False)
+    price = models.IntegerField(null=False)
+    description = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'products'
+
+class PurchaseMethod(models.Model):
+    name = models.CharField(max_length=30, null=False)
+    Product = models.ForeignKey(Product, on_delete=models.DO_NOTHING,null=False)
+
+    class Meta:
+        db_table = 'purchase_methods'
+
+class ProductPurchaseMethod(models.Model):
+    product = models.ForeignKey(Product,on_delete=models.DO_NOTHING, null=False)
+    purchase_method = models.ForeignKey(PurchaseMethod,on_delete=models.DO_NOTHING, null=False)
+
+    class Meta:
+        db_table = 'products_purchase_methods'
+
+class ProductImage(models.Model):
+    product_image = models.ImageField()
+
+    class Meta:
+        db_table = 'products_images'
```

Comment:
product image가 product를 FK로 참조하고 있어야하지 않을까요?
컬럼명도 verbose 하지 않고 간결하고 명확하게 만들어주세요~

```suggestion
class ProductImage(models.Model):
    product = models.ForeignKey(Product,on_delete=models.DO_NOTHING, null=False)
    image_url = models.ImageField()

    class Meta:
        db_table = 'product_images'
```